tests/test_types.py

```diff
@@ -132,6 +132,39 @@ def test_datetime_accepts_datetime():
     assert dt_type.to_primitive(dt) == output
 
 
+def test_timedelta():
+    type_ = TimedeltaType()
```

Comment:
I don't think assigning to an intermediary `type_` variable is necessary in these tests -- I see some other tests use an intermediary, but usually when the type is slow to init or many versions/init options are used, so as not to repeat them.